Data_Preprocessing/nbconverted/file_merge.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import pyarrow as pa
+import pyarrow.parquet as pq
+
+# In[2]:
+
+
+# define input paths
+first_run_sc_fs_path = Path(
+    "/media/lippincm/c58d4f19-ae4d-4b78-8370-2c2639886da0/Interstellar_Plate2/SHSY5Y_first_run_sc_norm_fs.parquet"
+)
+first_run_sc_path = Path(
+    "/media/lippincm/c58d4f19-ae4d-4b78-8370-2c2639886da0/Interstellar_Plate2/SHSY5Y_first_run_sc_norm.parquet"
+)
+
+second_run_sc_fs_path = Path(
+    "/media/lippincm/c58d4f19-ae4d-4b78-8370-2c2639886da0/Interstellar_Plate2/SHSY5Y_second_run_sc_norm_fs.parquet"
+)
+second_run_sc_fs_path = Path(
+    "/media/lippincm/c58d4f19-ae4d-4b78-8370-2c2639886da0/Interstellar_Plate2/SHSY5Y_second_run_sc_norm.parquet"
+)
```

Comment:
It looks like you overwrite the path here. Did you mean to do this? Probably not but if this is intentional consider explaining why with a comment.

Reply:
ah, these are separate paths, the first one is _fs and the second is not. I can document which is which to add clarity

Reply:
Never mind, the path is right but not the variable name... changing now